extensions/measurement-tracking/src/panels/PanelStudyBrowserTracking/PanelStudyBrowserTracking.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
-import { HangingProtocolService, utils } from '@ohif/core';
+import { HangingProtocolService, utils, ViewportGridService } from '@ohif/core';
```

Comment:
why are you importing `ViewportGridService` when it is not being used?

Reply:
will fix